File: AdotePetJavaProjetct/src/adotepet/Vacina.java
package adotepet;
import java.util.Date;

public class Vacina{
    private String nome;
    private Date dataVacinacao;

    public String getNome(){
        return nome;
    }
	
    public void setNome(String nome){
        this.nome=nome;
    }

    public Date getDataVacinacao(){
        return dataVacinacao;
    }
	
    public void setDataVacinacao(Date dataVacinacao){
        this.dataVacinacao=dataVacinacao;
    }
}

File: AdotePetWeb/backend/api/model/Vacina.php
<?php

/**
 * @author Eduardo
 */
class Vacina {
   
    private $nome;

    public function get_nome() {
        return $this->nome;
    }

    public function set_nome($nome) {
        $this->nome = $nome;
        return $this;
    }

}